source/api_documentation/network_integration/promo_numbers/_put_promo_number_by_id_with_id.rst

```diff
@@ -53,3 +53,35 @@
   Response Body:
 
   Same as a GET response, includes all the promo number properties.
+
+  .. raw:: html
+
+    <hr>
+
+  Moving an Advertiser Campaign Promo Number
+
+  **Note:** When a promo number is moved, its **id_from_network** will be automatically adjusted.
```

Comment:
❓ Are you **sure** this is always accurate?

What about if they provide an `id_from_network` within the request body?

Reply:
Dug as little to look into this and I believe so yes, but actually I think this might be something that can be fixed with little effort. 

I thought that there might have been some concern around having a duplicate id_from_network since that would break validation, but the move process deletes the old VL before saving the new; it just doesn't copy over the `id_from_network`.  I'll see if the lift is as easy as it looks...

Reply:
❓ So to clarify, if the user provides an `id_from_network` in the request body as part of the update, will it be respected and applied to the virtual line?

Reply:
Correct, it will honor a given `id_from_network` and, if it's not given, fall back to the promo numbers original `id_from_network`

```
previous_id_from_network = promo_number.id_from_network
...
# Update/move logic
...
promo_number.id_from_network = new_params[:id_from_network] || previous_id_from_network
```